ConanInstall.cmake

```diff
@@ -30,7 +30,7 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
 
   include(${CMAKE_BINARY_DIR}/conan.cmake)
 
-  conan_check(VERSION 1.48.0 REQUIRED)
+  conan_check(VERSION 1.52.0 REQUIRED)
```

Comment:
```suggestion
  conan_check(VERSION 1.53.0 REQUIRED)
```

Reply:
Great! Let's bump it to 1.53!

I believe we could merge the PR if you have time.
Looks like I cannot open the page for `Incremental ubuntu-22.04 Build - CMake failed`'s details. I don't think it will cause any issues though. 

Thanks,

Reply:
the jenkins log are restricted to collaborators. The failure is unrelated to this change.